src/ai/backend/kernel/intrinsic.py

```diff
@@ -148,9 +148,16 @@ async def prepare_sshd_service(service_info):
 
 async def prepare_ttyd_service(service_info):
     shell = "sh"
-    if Path("/bin/bash").exists():
+    if Path("/bin/zsh"):
+        shell = "zsh"
+        zshrc_path = Path("~/.zshrc").expanduser()
+        with zshrc_path.open("w") as f:
+            f.write("export PATH=$PATH:/opt/backend.ai/bin\n")
```

Comment:
Is this necessary? `/opt/backend.ai/bin` is intended to be private for our kernel runner. For instance, if the user executes "python" in their terminal sessions, it should launch the image's default Python instead of our own prebuilt Python tailored for the kernel runner.

Reply:
In that case you are right, this isn't necessary